Ginger/GingerCoreCommon/UIElement/ElementInfo.cs

```diff
@@ -259,6 +260,132 @@ public virtual string GetValue()
             return mValue;
         }
 
+
+
+
+
+
+
+        private bool mIsSelected = false;
```

Comment:
i'm thinking we should have wrapped "ElementInfo" with "DeltaElementInfo" new object which will have all ElementInfo inhrit fields and will add those Delta fields on top only for Delta Wizard needs instead messing ElementInfo with all of those extra fields which will be used only for Delta